avroExample/src/main/resources/compendium.postman_collection.json

```diff
@@ -0,0 +1,187 @@
+{
```

Comment:
I think this Postman collection might make more sense in the root of the project (inside a `postman` folder, for example), otherwise, it might be distributed along with the code

Reply:
Yes, put it in the root of the project makes sense to me